### PR TITLE
update instructions, remove images

### PR DESCRIPTION
**Pull request by nataliejeremy**
@rachelwhitton or @bmackinney please review. I logged in to my godaddy account to update the instructions.

### CODE REVIEW

**Comment by nataliejeremy** [comment]
@rachelwhitton Can you review and merge this?

File: source/docs/articles/sites/domains/setting-up-a-domain-with-godaddy.md [hunk 1]
```diff
@@ -1,44 +1,27 @@
...
+
+If you need to add email to your new domain, you can add it at this time. Pantheon does not provide email hosting services.
+
+After you've purchased your domain name, follow the instructions below.  
+
+## Configure Domain Settings
+
1. Sign in to your account at GoDaddy.com.
+2. Click **Manage My Domains**, then **Manage DNS**.
+3. Enter the **A record** or **CNAME** for your domain. Make sure to enter the IP address or the CNAME to point your domain to Pantheon. For more information, see [Going Live](/docs/articles/going-live/).
```
Comment (rachelwhitton) [inline source/docs/articles/sites/domains/setting-up-a-domain-with-godaddy.md]: These instructions don't seem to reflect current GoDaddy API.

First, you must click **Manage** in the Domains section:
screen shot 2015-07-20 at 5 09 33 pm
Then click the gears icon and select **Manage DNS**:
screen shot 2015-07-20 at 5 17 57 pm

We don't need screenshots in the doc, just added here for context. @nataliejeremy do you see the same thing when logging into your GoDaddy account?

Comment (nataliejeremy) [inline source/docs/articles/sites/domains/setting-up-a-domain-with-godaddy.md]: @rachelwhitton So you're saying it was missing the gear icon step? I thought that was obvious but we should add it just to be clear. thanks!

File: source/docs/articles/sites/domains/setting-up-a-domain-with-godaddy.md [hunk 2]
```diff
@@ -1,44 +1,27 @@
...
+If you need to add email to your new domain, you can add it at this time. Pantheon does not provide email hosting services.
+
+After you've purchased your domain name, follow the instructions below.  
+
+## Configure Domain Settings
+
1. Sign in to your account at GoDaddy.com.
+2. Click **Manage My Domains**, then **Manage DNS**.
+3. Enter the **A record** or **CNAME** for your domain. Make sure to enter the IP address or the CNAME to point your domain to Pantheon. For more information, see [Going Live](/docs/articles/going-live/). 
+
```
Comment (rachelwhitton) [inline source/docs/articles/sites/domains/setting-up-a-domain-with-godaddy.md]: I think we should state the recommended configurations:
>1. DNS CNAME Record for www subdomain
>2. DNS A Record for non-www domain
>3. Redirect non-www to www in settings.php or wp-config.php

Should we add **how** these records are created using GoDaddy?

Comment (nataliejeremy) [inline source/docs/articles/sites/domains/setting-up-a-domain-with-godaddy.md]: I know Brian will address this in the Domains doc update, but go ahead and add it here too.

File: source/docs/articles/sites/domains/setting-up-a-domain-with-godaddy.md [hunk 3]
```diff
@@ -1,44 +1,27 @@
...
 <div class="alert alert-warning" role="alert">
-<strong>Note</strong>: If you have SSL enabled on your site, you have been given a unique I.P. address. This is the I.P. that you should point the A record to.</div>  
- ![Configure the A records](/source/docs/assets/images/desk_images/49503.png)  
-### Add Your Domain to Pantheon
+<strong>Note</strong>: If you have SSL enabled on your site, you have been given a unique IP address. Point the A record to this IP address.</div>  
+
+## Add Your Domain to Pantheon
```
Comment (rachelwhitton) [inline source/docs/articles/sites/domains/setting-up-a-domain-with-godaddy.md]: Adding the domain to Pantheon should be done first - to generate the recommended DNS records that will need to be added at GoDaddy.

File: source/docs/articles/sites/domains/setting-up-a-domain-with-godaddy.md (hunk 1 above)
Comment (nataliejeremy) [inline source/docs/articles/sites/domains/setting-up-a-domain-with-godaddy.md]: And no, that's not how I got there. After signing in I'm taken to the dashboard where I can click Manage next to Domains.

Comment (rachelwhitton) [inline source/docs/articles/sites/domains/setting-up-a-domain-with-godaddy.md]: @nataliejeremy - where are you taken after you click that? 
Here's what I'm seeing:
1. Sign in to your GoDaddy account.
2. Click the **Manage** button within the Domains section.
3. Click on the gears icon for the domain you wish to configure, and select **Manage DNS**.
4. Create the appropriate **A record** and/or **CNAME** records for your domain (provided in the site dashboard).

**Comment by rachelwhitton** [comment]
+1 

I rewrote some instructions, @nataliejeremy can you review one last time and merge if it looks good?

**Pull request merged**